cachito/web/content_manifest.py

```diff
@@ -31,13 +37,14 @@
 class ContentManifest:
     """A content manifest associated with a Cachito request."""
 
-    def __init__(self, request):
+    def __init__(self, request, packages):
         """
         Initialize ContentManifest.
 
         :param Request request: the request to generate a ContentManifest for
```

Comment:
Missing description of argument `packages` in the docstring.

Reply:
FTFY :slightly_smiling_face: